core/templates/pages/learner-dashboard-page/learner-dashboard-page.component.ts

```diff
@@ -232,9 +232,16 @@ angular.module('oppia').component('learnerDashboardPage', {
         return value;
       };
 
+      var changeDelay = function() {
+        const delaydrag = 1000;
+        const instantdrag = 0;
```

Comment:
Hm, this is not quite what I meant. I meant something like this:

https://github.com/oppia/oppia/blob/fe3e910b096907611bc4b285da9415c59e473e03/core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts#L112-L115

Note that they contain units and are in caps.

Reply:
Maybe this?
DRAG_DELAY_MOBILE_MSEC
DRAG_DELAY_DESKTOP_MSEC

Reply:
DONE